content/applications/sales/crm/track_leads/prospect_visits.rst

```diff
@@ -1,55 +1,140 @@
-*Page Views* button but also further down you will see them in the
-chatter.
+When a lead is created from a prospective customer filling out a :ref:`contact form
+<crm/track_leads/webpage-visits-contact-form>` on the website, Odoo tracks every webpage visited by
+that contact. This information is available on the lead/opportunity form in the *CRM* app.
 
-Both will update if the viewers comes back to your website and visits
-more pages.
+To see a prospective customer's page views, navigate to their lead/opportunity form in the *CRM*
+app.
 
-.. image:: prospect_visits/prospect_visits04.png
+The :guilabel:`Page Views` smart button at the top of the opportunity form shows the total number of
+times the prospective customer viewed a page on the website (including multiple views of the same
+webpage). Click on the smart button to see an organized list of each individual page view.
+
+.. image:: prospect_visits/page-views-smart-button.png
    :align: center
+   :alt: Page views smart button on an opportunity form.
+
+If a prospective customer clicks on a tracked link, then fills out a contact form on the website,
+and the contact form is :ref:`configured to create a lead
+<crm/track_leads/webpage-visits-contact-form>` in Odoo, then the campaign, medium, and source
+information from the :ref:`link tracker <crm/track_leads/link-tracker>` appears on the lead form.
+
+This information appears in the :guilabel:`Marketing` section of the :guilabel:`Extra Information`
+tab of the lead or opportunity form.
 
-.. image:: prospect_visits/prospect_visits05.png
+.. image:: prospect_visits/campaign-medium-source.png
    :align: center
+   :alt: Campaign, medium, and source on an opportunity form.
+
+.. _crm/track_leads/webpage-visits-contact-form:
+
+Contact form configuration
+--------------------------
+
+To configure a website contact form that creates a lead or opportunity in the *CRM* app when it is
+filled out, navigate to the contact form page on the *Website* app, which is typically accessible
+via a :guilabel:`Contact Us` page in the header menu of the company website. Then, click the
+:guilabel:`Edit` button at the top-right of the page to open the website editor.
+
+Next, click on the existing contact form block on the webpage. Or, to add a new form block, scroll
+down to the :guilabel:`Dynamic Content` section of the :guilabel:`Blocks` tab in the website editor.
+Then drag-and-drop the :guilabel:`Form` block onto the body of the webpage.
+
+Click on any field of the form block on the webpage to reveal the :guilabel:`Customize` tab of the
+website editor on the right sidebar. In the :guilabel:`Form` section, set the :guilabel:`Action` to
+:guilabel:`Create an Opportunity`.
+
+With this configuration, submitting a filled-out contact form creates a lead in the
+:guilabel:`Leads` section of the *CRM* app. Or, if the :guilabel:`Leads` feature is **not**
+activated in the *CRM* settings, it creates an opportunity in the first stage of the pipeline.
```

Comment:
No need for :guilabel:s here since this text is not on the screen on the active UI, rather it is being generally referred to
```suggestion
With this configuration, submitting a filled-out contact form creates a lead in the *Leads* section
of the *CRM* app. Or, if the *Leads* feature is **not** activated in the *CRM* settings, it creates
an opportunity in the first stage of the pipeline.
```